src/bulkExport/types.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { ExportType } from 'fhir-works-on-aws-interface';
+import { JobRunState } from 'aws-sdk/clients/glue';
+
+/**
+ * Bulk export state machine parameters.
+ * All lambda functions in the state machine are expected to use this type as both input and output
+ */
+export interface BulkExportStateMachineGlobalParameters {
+    jobId: string;
+    exportType: ExportType;
+    transactionTime: string;
+    requestQueryParams?: {
```

Comment:
I think we can just move `_outputFormat`, `_since`, and `_type` to the root level of this interface.We don't need to nest it in `requestQueryParams`. In `export-request` table we store these value at the root level. Also in the interface it's the same.
https://github.com/awslabs/fhir-works-on-aws-interface/blob/bulk-export/src/bulkDataAccess.ts#L5

Reply:
updated